README.md

```diff
@@ -109,7 +124,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
+FSMDiagram.jpeg  
```

Comment:
# 3

The lines between states in the diagram represent events. In your diagram, what event gets me from S1 to S2? And how do I get from S4 to the END is there is no adverb (because if there's no input after the verb, then the event is EOI).